example/issue34.rb

```diff
@@ -0,0 +1,39 @@
+#!/bin/env ruby
```

Comment:
This is an example script to show usage of 256-color palette and 8-color (with mixing).